platform/marvell-armhf/sonic-platform-nokia/7215/sonic_platform/chassis.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python
```

Comment:
is the script python3 compatible? 

@jleveque , i think all platform modules should be python3 compatible now since we are switching to python3 now.